src/static/js/modules/form-explainer.js

```diff
@@ -434,8 +434,11 @@ $(document).ready(function(){
      itemID = $this.hasClass('expandable__form-explainer') ? $this.attr('id') : $this.parent('.expandable__form-explainer').attr('id'),
       $overlay = $('.image-map_overlay'),
       $target = $('a[href=#' + itemID + ']');
-    $overlay.removeClass('has-attention');
-    $target.addClass('has-attention');
+      $('.expandable__form-explainer').removeClass('hover-has-attention');
+    var $expandable = $this.hasClass('expandable__form-explainer') ? $this : $this.closest('.expandable__form-explainer');
+    $expandable.addClass('hover-has-attention');
+    $overlay.removeClass('hover-has-attention');
+    $target.addClass('hover-has-attention');
```

Comment:
this section's indent level is inconsistent, [our standard is 2 spaces](https://github.com/cfpb/front-end/blob/master/javascript.md#spacing) which is in our `.editorconfig`

check out how to make use of `.editorconfig` files in your text editor: http://editorconfig.org/#download